apple/testing/default_runner/ios_test_runner.template.sh

```diff
@@ -221,39 +225,37 @@ if [[ -n "${LAUNCH_OPTIONS_JSON_STR}" ]]; then
 fi
 
 target_flags=()
-if [[ -n "${REUSE_GLOBAL_SIMULATOR:-}" ]]; then
```

Comment:
i don't follow the logic change here, but this is intended for users to pass so removing this functionality is probably bad?